hw/ip/otbn/dv/model/otbn_model.h

```diff
@@ -90,6 +90,11 @@ class OtbnModel {
   // error. Returns 0 on success; -1 on failure.
   int invalidate_dmem();
 
+  // Tell the model to tolerate the next occurring mismatch between RTL and ISS
+  // trace entries. Required for tests covering FI countermeasures with delayed
+  // escalation.
+  int tolerate_result_mismatch();
```

Comment:
I don't think I can see any reason for this to have a return value. Can't it be `void`?